README.md

```diff
@@ -52,20 +54,24 @@ and logging.
 
 # Contact Information
 
-This charm is available at <https://jujucharms.com/docker> and contains the 
-open source operations code to deploy on all public clouds in the Juju 
+This charm is available at <https://jujucharms.com/docker> and contains the
+open source operations code to deploy on all public clouds in the Juju
 ecosystem.
 
 ## Docker links
 
   - The [Docker homepage](https://www.docker.com/)
-  - Docker [documentation](https://docs.docker.com/) for help with Docker 
+  - Docker [documentation](https://docs.docker.com/) for help with Docker
   commands.
   - Docker [forums](https://forums.docker.com/) for community discussions.
-  - Check the Docker [issue tracker](https://github.com/docker/docker/issues) 
+  - Check the Docker [issue tracker](https://github.com/docker/docker/issues)
   for bugs or problems with the Docker software.
   - The [charm-docker](https://github.com/juju-solutions/charm-docker) is
   the GitHub repository that contains the reactive code to build this Charm.
   - Check the charm-docker
   [issue-tracker](https://github.com/juju-solutions/charm-docker/issues) for
   bugs or problems related to the Charm.
+
+# Testing
+
+When upgrading this charm, the [`docker_runtime_version`](https://github.com/VariableDeclared/jenkins/blob/3bf2cabc7185568e6a80a137a7f083c428048683/jobs/validate-juju-https-envs.yaml#L22) of the `validate-https-proxy-envs` key needs to be updated with the new containerd version.
```

Comment:
I suggest removing this from the README and removing the need to hard-code charm revs in the test.

More detailed comment: https://github.com/charmed-kubernetes/charm-containerd/pull/8#discussion_r305098886